doc/sphinx/source/conf.py

```diff
@@ -157,6 +157,8 @@
 
 # -- Options for LaTeX output ---------------------------------------------
 
+latex_engine = 'xelatex'
```

Comment:
@jedbrown this latex engine on my machine produces the following error:
```
The font "FreeSerif" cannot be found.
```

Reply:
Ah, it uses that (an OTF font) by default, instead of the legacy LaTeX fonts. FreeSerif is a unicode extension of Times.
https://www.sphinx-doc.org/en/master/latex.html

What fonts would you prefer? We used Tex Gyre Pagella (a Palatino derivative) for the Jupyter book.

I'd prefer to switch the mono font to DejaVu Sans Mono or Fira Mono/Fira Code.

Reply:
I have installed the `fonts-freefont-otf` font package, but the latexpdf build still gives me errors.

PS: I would not use mono fonts for the main body of the text, since we often show code snippets, for which the mono-spaced font is appropriate

Reply:
I mean `\setmonofont` in `latex_elements`, e.g.,
```
    'fontenc': r'\usepackage{mathspec}',
    'fontpkg': r'''
\setmainfont{TeX Gyre Pagella}
\setmathfont{TeX Gyre Pagella Math}
\setsansfont{DejaVu Sans}
\setmonofont{DejaVu Sans Mono}
''',
```
Courier looks very old-fashioned in the code snippets.

Reply:
Ah yes, only for the `\setmonofont` yes, fine. I thought you were talking in general, about fonts for the main body. 

The latexpdf build is still not successful for me with the `xelatex` engine even though I semm to have installed all required packages.